<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>pic/user.png</normaloff>pic/user.png</iconset>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="emal_label">
      <property name="text">
       <string>邮箱：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="email_edit">
      <property name="placeholderText">
       <string comment="请输入邮箱账号" extracomment="请输入邮箱账号"/>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="password_label">
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="password_edit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string comment="请输入密码" extracomment="请输入密码"/>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="login_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pic/login.png</normaloff>pic/login.png</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="loadingLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>login_btn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>login_slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>login_slot()</slot>
 </slots>
</ui>
